README.md

```diff
@@ -61,6 +61,8 @@ use: "file-loader?name=[name].[ext]&publicPath=assets/foo/&outputPath=app/images
 }
 ```
 
+`cssOutputPath` also should be used (together with `useRelativePath` property) to define the output path context.
```

Comment:
hahah, ok, I'll make a sample folder and write some tests on it. I don't know if I finish today but I'll try. And, yes, I need to put the webpack and ETWP, also make some fixes on travis, maybe some `commits` to fix it or put engines on the package to fix the integration test. Is there any problem?